app/helpers/container_summary_helper.rb

```diff
@@ -71,6 +71,14 @@ def textual_container_images
     textual_link(@record.container_images)
   end
 
+  def textual_guest_applications
+    textual_link(@record.guest_applications, :feature => "containers",
```

Comment:
> @dkorn Is that the correct feature to have here? 
> should probably be container_image_scan or container_image_view I think...

Thanks for bringing this up, I added a new product feature: container_image_package_show_list